BlockingQueueSplit.tla

```diff
@@ -85,19 +87,19 @@ INSTANCE TLAPS
 LEMMA ITypeInv == Spec => []TypeInv
 <1> USE Assumption DEF TypeInv
 <1>1. Init => TypeInv
-  BY SMT DEF Init
+  BY DEF Init
 <1>2. TypeInv /\ [Next]_vars => TypeInv'
-  BY SMT DEF Next, vars, Put, Get, Wait, NotifyOther
+  BY DEF Next, vars, Put, Get, Wait, NotifyOther
 <1>3. QED
   BY <1>1, <1>2, PTL DEF Spec
 
 THEOREM Implements == Spec => A!Spec
 <1> USE Assumption, A!Assumption
 <1>1. Init => A!Init 
-   BY Isa DEF Init, A!Init
+   BY DEF Init, A!Init
 <1>2. TypeInv /\ [Next]_vars => [A!Next]_A!vars
-   BY SMT DEF TypeInv, Next, vars, Put, Get, Wait, NotifyOther, A!Next, 
-          A!vars, A!Put, A!Get, A!Wait, A!NotifyOther
+   BY DEF TypeInv, Next, vars, Put, Get, Wait, NotifyOther, A!Next, 
+          A!vars, A!Put, A!Get, A!Wait, A!NotifyOther, A!RunningThreads
```

Comment:
The proof goes through without `A!RunningThreads` on my macOS 13.4 with tlapm 1.4.5 (build 33809)

Reply:
Committed (force pushed!) to main:

https://github.com/lemmy/BlockingQueue/commit/06b6509f84e711bbb8ae144ca56fd99a7b26795b#diff-40e25cd036b51753a8c041bff8d94f186eea111f3f2fcf9a4246be8177e408b8R101